source/server.js

```diff
@@ -19,19 +19,28 @@ const Bluebird = require('bluebird');
 
 process.on('uncaughtException', (err) => {
   winston.error(err.stack ? err.stack.toString() : err.toString());
-  bugtracker.notify.bind(bugtracker, err, 'ungit-launcher');
+  bugtracker.notify(err, 'ungit-launcher');
```

Comment:
@jung-kim @FredrikNoren Is the raven (sentry.io) bugtracking still active or can it be removed?

This got broken two years ago https://github.com/FredrikNoren/ungit/commit/543449e16ac83e1635d300c4e66b9121536e815e#diff-71e7385fb3d187340c7741bf6d4f82ffL25-R27

Reply:
I'm not sure anymore as either I cannot find my access nor I no longer have access.

We could potentially open a new free account and use it that way as it is very possible that account is no longer active.